src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -879,12 +879,20 @@ void checkUnsafeDependencies()
         if ( CollectionUtils.isNotEmpty( unsafeDeps ) )
         {
             Log log = getLog();
-            log.warn( "There is " + unsafeDeps.size() + " dependencies with no license :" );
-            for ( MavenProject dep : unsafeDeps )
+            if ( log.isWarnEnabled() )
             {
+                boolean plural = unsafeDeps.size() > 1;
```

Comment:
If `unsafeDeps.size() == 0` the message will be "There is 0 dependency...". Clearly, "There are 0 dependencies..." would be better. Could you please change the condition accordingly?